packages/store/ts/config/v2/store.test.ts

```diff
@@ -557,11 +563,13 @@ describe("resolveStoreConfig", () => {
           },
         },
       }),
-    ).type.errors(`Type '"firstKey"' is not assignable to type '"secondKey" | "secondAge"'`);
+    )
+      .throws("Invalid primary key. Expected (secondKey|secondAge)[], received [firstKey,secondAge]")
```

Comment:
while we're here, can we wrap these key names in double quotes? and backticks around each expression?

> Invalid primary key. Expected `("secondKey" | "secondAge")[]`, received `["firstKey", "secondAge"]`.

I wanna get to a place where our errors rival viem!

Reply:
and would like to come back to this later and add more branches like 
> `Invalid primary key. `"firstKey"` is not a static ABI type and cannot be used as part of a key.
> `Invalid primary key. No field named `"firstKey"` in schema.